CONTRIBUTING.md

```diff
@@ -150,11 +149,13 @@ down to the following steps:
     existing commit, use common sense to decide which is better), they will be
     automatically added to the pull request.
 
- 7. If your pull request changes anything that users should be aware of (a
-    bugfix, a new feature, ...) please add an entry to the file
-    ['CHANGELOG.md'](CHANGELOG.md). It will be used in the announcement of the
-    next stable release. While writing, ask yourself: If I were the user, what
-    would I need to be aware of with this change.
+ 7. If your pull request changes anything that users should be aware
+    of (a bugfix, a new feature, ...) please add an entry in
+    `changelog/unreleased` using the name of the issue (`issue-8756`)
```

Comment:
It's unclear that it's the file that should be named like this. Perhaps "... including the issue number in the filename (e.g. `issue-8756`) ..."